t/node/healthcheck.t

```diff
@@ -906,3 +906,99 @@ qr/\([^)]+\) unhealthy .* for '.*'/
 (upstream#/apisix/upstreams/1) unhealthy TCP increment (1/2) for 'foo.com(127.0.0.1:1970)'
 (upstream#/apisix/upstreams/1) unhealthy TCP increment (2/2) for 'foo.com(127.0.0.1:1970)'
 --- timeout: 10
+
+
+
+=== TEST 20: set route(two healthy upstream nodes)
+--- config
```

Comment:
> When the test file is too large, for example > 800 lines, you should split it to a new file. Please take a look at t/plugin/limit-conn.t and t/plugin/limit-conn2.t.

https://github.com/apache/apisix/blob/master/CONTRIBUTING.md#check-code-style-and-test-case-style

Reply:
Done